### PR TITLE
Build /data/mongodb

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built page as per [doc](https://docs.google.com/document/d/1mCztOFZYbC3TeUM7j7pGfSoxKzBs20W1pJHZhdLqKs8/edit) (no mock-up)

## Notes

I had to make some guesses as to where to put some of the images, the isometric one in the hero I already know will probably need to be moved, but I'll wait for designs instructions to tell me where. As with the rest of my prs for this bubble, please note that any CTA that links to the modal will not work. Main CTA ("Get our guide to MongoDB security and support") is also missing as the datasheet it's supposed to link to is not live yet.

## QA

- View the site locally in your web browser at: https://canonical-com-1046.demos.haus/data/mongodb
- Check against doc

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5879

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
(@mtruj013 not a full review, doing this in between sessions at the sprint, you may want to wait for the full thing):
Please remove this:
image

Please do something like this for the hero spacing. I think the h2 exceeds the maximum word count in @juanruitina's content guide, so I would either reword, or restyle as a paragraph:
```
<div class="col p-strip is-shallow u-no-padding--bottom">
      <div class="p-section--shallow">
        <h1 class="p-section--shallow">MongoDB<sup>®</sup>database operations, simplified</h1>
        <p class="">Secure and automate the deployment, maintenance and upgrades of your MongoDB databases across private and public clouds. Deliver large amounts of data in high-performance applications.</p>
      </div>
      <hr>
      <a class="p-button--positive" href="">Get in touch</a>
      <a href="https://assets.ubuntu.com/v1/b5aa027c-MongoDB%20-%20Datasheet%20v2.pdf">Download the datasheet&nbsp;›</a>
    </div>
```

image

**Comment by lyubomir-popov** [comment]
please update to latest vanilla - this should be bolder in it:
image

**Comment by lyubomir-popov** [comment]
the aspect ratio here should be 3:2:
image

**Comment by lyubomir-popov** [comment]
the last p-section on all pages should be deep, to insert more space before the footer:

**Comment by lyubomir-popov** [comment]
I would do this section like this - fits better ewith the surrounding content. cc @juanruitina hope this doesn't break anything important from a UX perspective.

image
image

**Comment by mtruj013** [comment]
> This is it - end of review, and sorry for the piecemeal.

@lyubomir-popov No need to apologize! Honestly just being explicitly told that it's coming in pieces and also making it clear when the review is finished is super helpful and pretty much addresses any issues I have with piecemeal reviews 🙂

**Comment by mtruj013** [comment]
@lyubomir-popov ready for another look, lmk if I missed something

File: templates/data/mongodb.html
Comment (pedoch) [inline templates/data/mongodb.html:185]:
```suggestion
        <h3 class="p-heading--5">Web and mobile applications</h3>
```

I'd say make these `h3` since they are headings, for better accessibility and hierarchy. But maybe it's something UX should look at.

File: templates/data/mongodb.html
Comment (pedoch) [inline templates/data/mongodb.html:190]:
```suggestion
        <h3 class="p-heading--5">Internet of Things (IoT)</h3>
```

File: templates/data/mongodb.html
Comment (pedoch) [inline templates/data/mongodb.html:195]:
```suggestion
        <h3 class="p-heading--5">Logging and monitoring</h3>
```

File: templates/data/mongodb.html
Comment (pedoch) [inline templates/data/mongodb.html:242]:
```suggestion
          <h3 class="p-heading--5">Managed MongoDB</h3>
```

File: templates/data/mongodb.html
Comment (pedoch) [inline templates/data/mongodb.html:261]:
```suggestion
          <h3 class="p-heading--5">Consulting</h3>
```

**Review by pedoch — APPROVED** [review]
Thanks @mtruj013, great work! Just some super tiny comments.

**Comment by pedoch** [comment]
Quick question will the contact form be added before the page is merged?

**Comment by juanruitina** [comment]
Nice work, some comments:

- Turn "Run MongoDB on" into an H2 (keep the formatting)
- Please add alt text to the logos: Microsoft Azure, Kubernetes, OpenStack, VMware, Google Cloud, Amazon Web Services
- Can you add a bit of spacing at the bottom of the image under "A hyper-automated MongoDB with enterprise-grade support" on mobile? It's too close to the list below.
- Remove the hyphenation in "MongoDB enterprise use cases".
- Turn the bold items under "MongoDB enterprise use cases" and "Services" into H3s

I noticed we messed up a bit the logos in the wireframes and mockups, sorry for this. We should use the original [OpenStack](https://assets.ubuntu.com/v1/95212c08-OpenStack_Logo_2016.svg) and [Ceph](https://assets.ubuntu.com/v1/e0461e14-ceph-dark.svg) logos, not Canonical's, all across the bubble, as per the copy docs (it also makes more sense UX wise). Not sure if visual needs to generate a logo cloud ready version of these.

**Comment by mtruj013** [comment]
@pedoch @juanruitina thanks both! I've addressed your comments.

To answer a couple of questions:

> will the contact form be added before the page is merged?

This PR is being merged into a feature branch, not main, so I'm not sure at what stage the modal will be added. I assume it'll be in a follow up PR if this one is merged before the modal is (pr for that [here](https://github.com/canonical/canonical.com/pull/1043)), but I leave that decision up to @carkod . Basically to answer your question, yes all CTAs will be updated before the feature branch gets merged into main 🙂 

> Not sure if visual needs to generate a logo cloud ready version of these.

I think so, at least this version doesn't size correctly. cc: @camillet16 
image

**Comment by camillet16** [comment]
@mtruj013 oh no we're already using canonical openstack. Not the OG openstack logo, feel free to remove.

**Comment by lyubomir-popov** [comment]
All the logos here are blurred - any idea why?

**Comment by carkod** [comment]
> All the logos here are blurred - any idea why?

They don't look blurred to me. It may take some time to load if you have a slow connection, it's the reason why we load them with cloudinary. So the page can load faster.

Screenshot 2023-09-28 at 15 25 48

**Comment by carkod** [comment]
> @camillet16 In case it wasn't clear, we _should_ use the original OpenStack and Ceph logos in this context, not Canonical's. This is in line with [Gloria's comment here](https://docs.google.com/document/d/1r1xnNihaTphO_eikifUEO7ocYfTDnbuRIaIfUCmya1c/edit?disco=AAAA6ExD0ew), and we do this on ubuntu.com too ([OpenStack](https://ubuntu.com/openstack), [Ceph](https://ubuntu.com/ceph)). MongoDB is an exception due to copyright restrictions.
> 
> Camille, can you provide the two assets?

Updated Openstack logo.

**Comment by juanruitina** [comment]
"Get in touch" and "Contact us" CTAs don't work yet, @carkod please check before launch.

**Comment by lyubomir-popov** [comment]
the logos are very very blurred: 
image

**Comment by carkod** [comment]
> the logos are very very blurred (only visible on high dpi 4k screens and above. on a tiny laptop, it is probably far less noticeable.) image

Well if you look at it this way, every single image on https://www.amazon.co.uk/ looks blurry... I don't think this should be a blocker, Gloria is actually saying that the hover effect is not visible for logos, so I think most people notice other things.

**Comment by lyubomir-popov** [comment]
@carkod I've re-exported the logos to try and fix the blurryness. But I have a feeling it is the cdn that is donwgrading them. Can you please try with these new ones, and provide exactly 3x the original size as parameters for each width and height? https://assets.ubuntu.com/manager?tag=mongodb-logos

**Comment by carkod** [comment]
> @carkod I've re-exported the logos to try and fix the blurryness. But I have a feeling it is the cdn that is donwgrading them. Can you please try with these new ones, and provide exactly 3x the original size as parameters for each width and height? https://assets.ubuntu.com/manager?tag=mongodb-logos

Done

**Comment by lyubomir-popov** [comment]
> Done

much better! We should export all logos at 3x I guess

**Comment by carkod** [comment]
@juanruitina modal fixed, can you take a look?

**Comment by juanruitina** [comment]
The "Contact us" button under Managed MongoDB links to the Canonical main contact us page, should show the modal for the MongoDB form instead. Can you fix?

Other than that, looks good to me. Feel free to change label to UX+1 once you fix.

**Pull request merged**
